neuralmonkey/config/parsing.py

```diff
@@ -143,84 +142,49 @@ def parse_value(string):
 
     return string
 
+def parse_config(config_file, filename=""):
```

Comment:
Ruší to ty globální proměnné. Jinak nevím o tom, že by to něco měnilo. Testy běží, ale nevím jestli to nerozbije něco dalšího, co je implementované mimo specifikaci a není použité v testovacím INI.